cmd/controller/certcontroller.go

```diff
@@ -73,11 +74,21 @@ var certcontrollerCmd = &cobra.Command{
 			}
 		}
 
+		// Configure metrics server options
+		metricsServerOpts := server.Options{
+			BindAddress: metricsAddr,
+		}
+
+		// Configure TLS options for metrics server
+		var tlsOpts []func(*tls.Config)
+		if !enableHTTP2 {
+			tlsOpts = append(tlsOpts, disableHTTP2)
+		}
+		metricsServerOpts.TLSOpts = tlsOpts
```

Comment:
```suggestion
		if !enableHTTP2 {
		    metricsServerOpts.TLSOpts = []func(*tls.Config){disableHTTP2}
		}
```
is this not possible?

Reply:
one can always refactor for that later without issues so we should keep code as simple as possible IMO. mostly a nit though would be cool if you could adjust. Thanks!